Messaging.md

```diff
@@ -169,52 +207,44 @@ The type `rounding_intervals` is defined [here](NumericOutcome.md#rounding-inter
 
 This type contains information about the oracles to be used in executing a DLC.
 
-#### Version 0 `oracle_info`
+#### `single_oracle_info`
 
-1. type: 42770 (`oracle_info_v0`)
-2. data:
+1. implements: `oracle_info`
+1. type: 0
+1. data:
    * [`oracle_announcement`:`oracle_announcement`]
 
 This type of oracle info is for single-oracle events.
 
-#### Version 1 `oracle_info`
+#### `multi_oracle_info`
 
-1. type: 42786 (`oracle_info_v1`)
-2. data:
+1. implements: `oracle_info`
```

Comment:
super nit: the `1`,`2`,`3`..

Reply:
This should render as 1 2 3, it's just easier to put only ones so you don't have to modify everything if you want to add something at the begging